tests/TestCase.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Letsgoi\LaravelSettings\Tests;
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Cache;
+use Orchestra\Testbench\TestCase as BaseTestCase;
+
+class TestCase extends BaseTestCase
+{
+    protected function setUp(): void
+    {
+        parent::setUp();
+    }
+
+    protected function tearDown(): void
+    {
+        Cache::flush();
+
+        parent::tearDown();
+    }
+
+    protected function defineEnvironment($app)
```

Comment:
Can you add return type in this protected methods?

Reply:
added